src/content/configuration/output.md

```diff
@@ -903,10 +903,10 @@ The output will be:
     module.exports = factory();
   else if(typeof define === 'function' && define.amd)
     define([], factory);
-  else {
-    var a = factory();
-    for(var i in a) (typeof exports === 'object' ? exports : root)[i] = a[i];
-  }
+  else if(typeof exports === 'object')
+    Object.assign(exports, factory());
+  else
+    Object.assign(root, factory());
 })(typeof self !== 'undefined' ? self : this, function() {
```

Comment:
Consistent with the structure of the code of named `MyLibrary`. and `Object.assign` is simpler.

Reply:
See the reason above.